libvast/src/schema.cpp

```diff
@@ -254,6 +253,8 @@ get_schema_dirs(const caf::actor_system_config& cfg,
   return result;
 }
 
+using symbol_buffer = shared_schema_parser::symbol_buffer;
```

Comment:
```suggestion
using symbol_table = shared_schema_parser::symbol_buffer;
```

That would be the canonical name for this thingy.